common/culture/cultures/wc_highborne.txt

```diff
@@ -45,11 +70,20 @@ high_elf = {
 blood_elf = {
 	color = rgb { 255 51 0 }
 	
+	parents = { high_elf }
+	created = 2.1.1
+
 	ethos = ethos_stoic
 	heritage = heritage_highborne
 	language = language_thalassian
-	martial_custom = martial_custom_male_only
+	martial_custom = martial_custom_equal
 	traditions = {
+		tradition_republican_legacy
```

Comment:
Explain an idea behind this.

Reply:
Sin'dorei have refrained from crowning any heirs to the throne, prefering a regency, despite there existing potential claimants from the Sunstrider bloodline.

Reply:
The regency doesn't sound like a republic. Plus, I guess the legacy assumes it has a long history of republics while the blood elves haven't had them.

Reply:
Updated with `Religion Blending`.